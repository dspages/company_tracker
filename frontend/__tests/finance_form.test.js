import React from 'react';
import FinanceForm from '../components/finance/finance_form.jsx';
import renderer from 'react-test-renderer';


test('Component correctly renders', () => {
  const component = renderer.create(<FinanceForm
    />);
  let tree = component.toJSON();
  expect(tree).toMatchSnapshot();
});
