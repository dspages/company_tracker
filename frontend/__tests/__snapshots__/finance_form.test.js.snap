// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Component correctly renders 1`] = `
<div>
  <form
    className="new-form"
  >
    <div
      className="new-form-row"
    >
      Finance Form
      <label>
        Fiscal Year:
        <input
          onChange={[Function]}
          value={undefined}
        />
      </label>
      <label>
        Valuation:
        <input
          onChange={[Function]}
          value={undefined}
        />
      </label>
      <label>
        Assets:
        <input
          onChange={[Function]}
          value={undefined}
        />
      </label>
      <label>
        Liabilities:
        <input
          onChange={[Function]}
          value={undefined}
        />
      </label>
      <label>
        Gross Income:
        <input
          onChange={[Function]}
          value={undefined}
        />
      </label>
      <label>
        Expenses:
        <input
          onChange={[Function]}
          value={undefined}
        />
      </label>
      <button
        onClick={[Function]}
      >
        Confirm
      </button>
    </div>
  </form>
</div>
`;
